### PR TITLE
Move to alpha features

### PR DESCRIPTION
**Pull request by jotterbach**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,11 +24,17 @@
...
-    packages=["grove", "grove.pyqaoa", "grove.pyvqe", "grove.qft",
-              "grove.phaseestimation", "grove.teleport",
-              "grove.deutsch_jozsa", "grove.arbitrary_state",
-              "grove.bernstein_vazirani", "grove.simon",
-              "grove.amplification"],
+    packages=["grove",
+              "grove.pyqaoa",
+              "grove.pyvqe",
+              "grove.qft",
+              "grove.alpha.phaseestimation",
```
Comment (ampolloreno) [inline setup.py]: Do you want `grove.alpha` here?

Comment (ampolloreno) [reply setup.py]: You're also missing `fermion_transforms` (although I see it wasn't there to begin with) in this list.

**Comment by jotterbach** [comment]
Good eye @ampolloreno. We need to fix these explicit package imports anyways, but for now this is a good idea!

**Comment by ampolloreno** [comment]
Fixes #67  (Does it auto close if I just comment?)

**Pull request merged**